File: 1._A._Latihan_Random_Coding/Coding_JavaScript/1._Input_Data_Profil.js
// Input Data Profil dalam Program Sederhana di JavaScript


console.log ("\n Input Data Profil dalam Program Sederhana di JavaScript \n")


var nama = prompt ("Masukkan nama : ")
var kelas = prompt ("Masukkan kelas :")
var absen = Number (prompt ("Masukkan nomor absen :"))
var sekolah = prompt ("Masukkan sekolah :")
var tinggi = prompt ("Masukkan tinggi badan :")
var berat =  prompt ("Masukkan berat badan :")
var coding = prompt ("Masukkan coding kamu : ")
var lomba = prompt ("Masukkan lomba yang pernah kamu ikuti : ")
var darah = prompt ("Masukkan golongan darah kamu : ")
var hobi = prompt ("Masukkan hobi kamu : ")


var detail = `

- Nama           : ${nama}
- Kelas          : ${kelas}
- Nomor Absen    : ${absen}
- Sekolah        : ${sekolah}
- Tinggi Badan   : ${tinggi}
- Berat Badan    : ${berat}
- Coding         : ${coding}
- Lomba          : ${lomba}
- Golongan darah : ${darah}
- Hobi           : ${hobi}

`

console.log (detail)


console.log ("\n --- Batas --- \n")




console.log ("\n Hasil Logika Sederhana \n")


var hasil = Number (prompt ("Masukkan logic 1 : ")) 


if (hasil > 80) {
    console.log (`Lulus dong, skor = ${hasil}`)
}

else if (hasil < 50) {
    console.log (`Belum dong, skor = ${hasil}`)
}

else {
    console.log (`Jangan dulu, skor = ${hasil}`)
}


console.log ("\n --- Batas --- \n")




console.log ("\n Hasil Logika 1 \n")


var logic = Number (prompt ("Masukkan logic 2 : "))


if (logic > 5) {
    console.log (`Oke dong, logic = ${logic}`)
}

else {
    console.log (`Belum dong, logic = ${logic}`)
}


console.log ("\n --- Batas --- \n")




console.log ("\n Logika Rapor Sederhana \n")


var rapor = Number (prompt ("Masukkan nilai rapor : "))


if (rapor >= 90) {
    console.log (`A, skor = ${rapor}`)
}

else if (rapor >= 80) {
    console.log (`B, skor = ${rapor}`)
}

else if (rapor >= 70) {
    console.log (`C, skor = ${rapor}`)
}

else if (rapor >= 60) {
    console.log (`D, skor = ${rapor}`)
}

else if (rapor >= 50) {
    console.log (`E, skor = ${rapor}`)
}

else if (rapor >= 40) {
    console.log (`F, skor = ${rapor}`)
}

else {
    console.log (`Jelek amat, skor = ${rapor}`)
}


console.log ("\n --- Batas --- \n")




console.log ("\n Percabangan Nested \n")


var tes = Number (prompt ("Masukkan tes 1 :"))
var cek = true

if (cek) {
    if (tes > 5) {
        console.log (`Oke, tes = ${tes}`)
    }
    
    else {
        console.log (`Belum boleh dong, tes = ${tes}`)
    }
}

else {
    console.log (`Jangan dulu dong, tes = ${tes}`)
}


console.log ("\n --- Batas --- \n")




console.log ("\n Pendaftaran join JKT48 \n")


var usia = Number (prompt ("Masukkan usia kamu untuk Join JKT48 : "))
var cek = true


if (cek) {
    if ((usia >= 13) && (usia <= 19)) {
        console.log (`Boleh join JKT48, usia = ${usia}`)
    }
    
    else if (usia > 19) {
        console.log (`Sudah lebih dari cukup untuk join JKT48, usia = ${usia}`)
    }
    
    else {
        console.log (`Belum boleh ikut join JKT48, usia = ${usia}`)
    }
}

else {
    console.log (`Gak lolos join JKT48, usia = ${usia}`)
}


console.log ("\n --- Batas --- \n")




console.log ("\n Usia Produktif Manusia \n")


var usia = Number (prompt ("Masukkan usia kamu : "))
var cek = true


if (cek) {
    if ((usia >= 15) && (usia <= 64)) {
        console.log (`Usia produktif manusia, usia = ${usia}`)
    }
    
    else if (usia > 64) {
        console.log (`Sudah tua, usia = ${usia}`)
    }
    
    else {
        console.log (`Belum masuk usia produktif, usia = ${usia}`)
    }
}

else {
    console.log (`Masih kecil usianya, usia = ${usia}`)
}


console.log ("\n --- Batas --- \n")




console.log ("\n For dasar \n")


for (a = 0; a < 11; a++) {
    console.log (`Urutan ke - ${a}`)
}


console.log ("\n --- Batas --- \n")




for (b = 0; b < 11; b++) {
    console.log (`Urutan ke - ${b}`)
}


console.log ("\n --- Batas --- \n")




for (c = 0; c < 11; c++) {
    console.log (`Urutan ke - ${c}`)
}


console.log ("\n --- Batas --- \n")




console.log ("\n While dasar \n")


var a = 5

while (a < 11) {
    console.log (`Urutan ke - ${a}`)
    a++
}


console.log ("\n --- Batas --- \n")




var b = 10

while (b < 16) {
    console.log (`Urutan ke - ${b}`)
    b++
}


console.log ("\n --- Batas --- \n")




var c = 15

while (c < 21) {
    console.log (`Urutan ke - ${c}`)
    c++
}


console.log ("\n --- Batas --- \n")

File: 1._A._Latihan_Random_Coding/Coding_Python/1._Input_Data_Profil.py
# Input Data Profil dalam Program Sederhana di Python


print ("\n Input Data Profil dalam Program Sederhana  di Python \n")


nama = input ("Masukkan nama : ")
kelas = input ("Masukkan kelas :")
absen = int (input ("Masukkan nomor absen :"))
sekolah = input ("Masukkan sekolah :")
tinggi = input ("Masukkan tinggi badan :")
berat =  input ("Masukkan berat badan :")
coding = input ("Masukkan coding kamu : ")
lomba = input ("Masukkan lomba yang pernah kamu ikuti : ")
darah = input ("Masukkan golongan darah kamu : ")
hobi = input ("Masukkan hobi kamu : ")


detail = f"""

- Nama           : {nama}
- Kelas          : {kelas}
- Nomor Absen    : {absen}
- Sekolah        : {sekolah}
- Tinggi Badan   : {tinggi}
- Berat Badan    : {berat}
- Coding         : {coding}
- Lomba          : {lomba}
- Golongan darah : {darah}
- Hobi           : {hobi}


"""

print (detail)


print ("\n --- Batas --- \n")




print ("\n Hasil Logika Sederhana \n")


hasil = int (input ("Masukkan logic 1 : "))


if hasil > 70:
    print (f"Lulus dong, skor = {hasil}")
    
elif hasil < 50:
    print (f"Belum dong, skor = {hasil}")
    
else:
    print (f"Jangan dulu, skor = {hasil}")
    
    
print ("\n --- Batas --- \n")




print ("\n Hasil Logika 1 \n")


logic = int (input ("Masukkan logic 2 :"))


if logic > 5:
    print (f"Oke dong, logic = {logic}")
    
else:
    print (f"Belum dong, logic = {logic}")
    
    
print ("\n --- Batas --- \n")




print ("\n Logika Rapor Sederhana \n")


rapor = int (input ("Masukkan nilai rapor : "))


if rapor >= 90:
    print (f"A, skor = {rapor}")
    
elif rapor >= 80:
    print (f"B, skor = {rapor}")
    
elif rapor >= 70:
    print (f"C, skor = {rapor}")
    
elif rapor >= 60:
    print (f"D, skor = {rapor}")
    
elif rapor >= 50:
    print (f"E, skor = {rapor}")
    
elif rapor >= 40:
    print (f"F, skor = {rapor}")
    
else:
    print (f"Jelek amat, skor = {rapor}")
    
    
print ("\n --- Batas --- \n")




print ("\n Percabangan Nested \n")


tes = int (input ("Masukkan tes 1 :"))
cek = True


if cek:
    if tes > 5:
        print (f"Oke, tes = {tes}")
        
    else:
        print (f"Belum boleh dong, tes = {tes}")
        
else:
    print (f"Jangan dulu dong, tes = {tes}")
    
    
print ("\n --- Batas --- \n")




print ("\n Pendaftaran join JKT48 \n")


usia = int (input ("Masukkan usia kamu untuk Join JKT48 : "))
cek = True


if cek:
    if usia >= 13 and usia <= 19:
        print (f"Boleh join JKT48, usia = {usia}")
        
    elif usia > 19:
        print (f"Sudah lebih dari cukup untuk join JKT48, usia = {usia}")
        
    else:
        print (f"Belum boleh ikut join JKT48, usia = {usia}")
        
else:
    print (f"Gak lolos join JKT48, usia = {usia}")
    
    
print ("\n --- Batas --- \n")




print ("\n Usia Produktif manusia \n")


usia = int (input ("Masukkan usia kamu : "))
cek = True

if cek:
    if usia >= 15 and usia <= 64:
        print (f"Usia produktif manusia, usia = {usia}")
        
    elif usia > 64:
        print (f"Sudah tua, usia = {usia}")
        
    else:
        print (f"Belum masuk usia produktif, usia = {usia}")
        
else:
    print (f"Masih kecil usianya, usia = {usia}")


print ("\n --- Batas --- \n")




print ("\n For dasar \n")


for a in range (11):
    print (f"Urutan ke - {a}")
    
    
print ("\n --- Batas --- \n")




for b in range (11):
    print (f"Urutan ke - {b}")
    
    
print ("\n --- Batas --- \n")




for c in range (11):
    print (f"Urutan ke - {c}")
    
    
print ("\n --- Batas --- \n")




print ("\n While dasar \n")


a = 5

while a < 11:
    print (f"Urutan ke - {a}")
    a = a + 1
    
    
print ("\n --- Batas --- \n")




b = 10 

while b < 16:
    print (f"Urutan ke - {b}")
    b = b + 1
    
    
print ("\n --- Batas --- \n")




c = 15

while c < 21:
    print (f"Urutan ke - {c}")
    c = c + 1
    
    
print ("\n --- Batas --- \n")
